nuxt.config.js

```diff
@@ -85,7 +77,7 @@ module.exports = {
 	 */
 	axios: {
 		// See https://github.com/nuxt-community/axios-module#options
-		baseUrl: process.env.BASE_URL || "https://localhost:3030",
+		baseUrl: process.env.API_URL || "https://localhost:3030",
```

Comment:
```suggestion
		baseUrl: process.env.API_HOST || "https://localhost:3030",
```
I guess you mean `API_HOST` which is specified in the documentation.
https://vuepress.test.schul-cloud.org/0-GettingStarted/1-Build.html#nuxt-client